city-infrastructure-platform/settings.py

```diff
@@ -61,7 +61,9 @@
 
 # Application definition
 DJANGO_APPS = [
-    "django.contrib.admin",
+    "helusers",
+    "social_django",
```

Comment:
In the previous commit you added this, but now you are moving it. This makes the history a bit harder to read in the future, so why not do the line change in the earlier commit?

Reply:
It was actually removed and added again in the same commit, because helusers-packages needs to be in introduced in certain order relative to other Django-packages in order to work properly. But I will fix all these helusers-imports to be in a single commit.